README.md

```diff
@@ -2,4 +2,11 @@
 
 [![Build Status](https://travis-ci.org/githubuser/bahug.png)](https://travis-ci.org/githubuser/bahug)
 
-TODO Description.
+To run:
+
+    stack build && stack exec bahug
+
+To see splices:
+
+    stack clean && stack build --ghc-options=-ddump-splices
+    find . -name "*.dump-splices" -exec cat {} \;
```

Comment:
If you add `-dsuppress-all`, GHC won't fully qualify all the names, which can make it easier for beginners to read.